### PR TITLE
 relinted to match cookiecutter 

### PR DESCRIPTION
**Pull request by mrmcwethy**
#6 Needed more work

### CODE REVIEW

**Comment by mrmcwethy** [comment]
@tannewt Is this better?  I remove the docs directory here too. added conf.py, requirements.txt, readthedocs.yml  and api.rst.  reworked the comments in all the functional code.  Updated README.rst. Removed setup.py

File: README.rst [hunk 1]
```diff
@@ -1,5 +1,15 @@
-Display drivers for Adafruit CircuitPython.
-=========================================
+Introduction
+============
+
+.. image:: https://readthedocs.org/projects/adafruit-circuitpython-rgb_display/badge/?version=latest
+
+    :target: https://circuitpython.readthedocs.io/projects/rgb_display/en/latest/
+
+    :alt: Documentation Status
```
Comment (tannewt) [inline README.rst]: I think these extra lines make it invalid. The image, target and alt lines should be next to one another.

Comment (mrmcwethy) [reply README.rst]: ok

Comment (tannewt) [reply README.rst]: Yeah, I think there is a bug with it.

**Comment by mrmcwethy** [comment]
@tannewt i will create a release for this.

**Pull request merged**